packages/lesswrong/components/ea-forum/users/EAUsersProfile.tsx

```diff
@@ -462,13 +469,22 @@ const EAUsersProfile = ({terms, slug, classes}: {
           <EAUsersProfileImage user={user} />
           <Typography variant="headline" className={classNames(classes.username, {[classes.deletedUsername]: user.deleted})}>
             {username}{user.deleted && <span className={classes.accountDeletedText}>(account deleted)</span>}
-            {showDonatedIcon(user) &&
+            {showDonatedFlair(user) &&
+              <LWTooltip
+                placement="bottom-start"
+                title={`Donated to the Donation Election`}
```

Comment:
"Donation Election fund" => "Donation Election". There was a discussion on slack about capitalising "fund", but then I thought there were too capital letters when I tried that so I just removed it